python/src/pyripherals/peripherals/DDR3.py

```diff
@@ -602,7 +602,7 @@ def deswizzle(self, d, convert_twos=True):
 
         return chan_data
 
-    def data_to_names(self, chan_data, old=False):
+    def data_to_names(self, chan_data, bitfile_version=self.fpga.bitfile_version):
```

Comment:
I'm not certain how the default parameter self.fpga.bitfile_version will behave if DDR3 is instantiated before the FPGA is initialized. The preferred usage might be that the bitfile_version is determined from the h5 file. Doing so will allow for offline data analysis to be done without an FPGA instance.

Reply:
The code has been updated, as outlined below. The idea is that we will pass in the bitfile version number from the .h5 file when we call `data_to_names`. This keeps `data_to_names` indepedent from .h5 files, so that we keep the option of reading DDR data without saving to a .h5 file open.